### PR TITLE
added banner as an option now along with styleguide component

### PR DESCRIPTION
**Pull request by tomkrupka**
via - https://waynedotedu.tpondemand.com/entity/60662

image

### CODE REVIEW

File: resources/views/partials/menu-top.blade.php [hunk 1]
```diff
@@ -5,13 +5,17 @@
...
+            <div class="small-12 columns position-relative">
                 <div class="vertical-centering title-area">
                     <h1>
-                        <a href="/">
-                            @if(config('app.sub_title') !== null)<span>{{ config('app.sub_title') }}</span>@endif
-                            {{ $site['title'] }}
-                        </a>
+                        <div>
+                            <a href="/">
+                                @if($site['id'] !== 1669)
```
Comment (nickdenardis) [inline resources/views/partials/menu-top.blade.php]: Was this area meant to be changed? 

The `@if(config('app.sub_title') !== null)` check should still work if there is going to be any change to this area.

File: resources/views/partials/menu-top.blade.php [hunk 2]
```diff
@@ -30,6 +34,11 @@
                         </div>
                     </div>
                 </div>
+
+                @if(isset($banner) && $banner != false)
```
Comment (nickdenardis) [inline resources/views/partials/menu-top.blade.php]: If this is the trigger for the banner perhaps we add to the [`README.md`](https://github.com/waynestate/base-site/blob/master/README.md#developing-views) on how to trigger it from a controller?

Comment (robertvrabel) [reply resources/views/partials/menu-top.blade.php]: Its triggered via per page promotions in the `Banner` group defined in the `PromoRepository` -> https://github.com/waynestate/base-site/pull/23/files/bc7aacab6ce16520b316a1811729193728d11359#diff-6bd8c85afde6cd73327d6415a509969aR71

So we don't need to trigger it from a controller at all. I'm cool with expanding the documentation though and explaining how these global promotion groups work!

File: resources/views/partials/menu-top.blade.php (hunk 1 above)
Comment (tomkrupka) [reply resources/views/partials/menu-top.blade.php]: No it wasn't I will fix thanks!!

File: resources/views/partials/menu-top.blade.php (hunk 2 above)
Comment (nickdenardis) [reply resources/views/partials/menu-top.blade.php]: Sounds good. I think documenting all the inherent features of the overall template would be a good next step.

**Pull request merged**